CHANGELOG.md

```diff
@@ -1,7 +1,32 @@
 **2.0.0 - to be released**
 
 * Convert plugin to Kotlin
```

Comment:
Let's add these too:
- Add group tasks to publish all variants
- Unknown signing configs no longer fail the build
- Improve error messages
- Add status indicators and upload progress